src/translations/en.ts

```diff
@@ -19,4 +19,13 @@ export default {
     'file-viewer.open-in-fullscreen': 'Open in fullscreen',
     'file-viewer.open-in-new-tab': 'Open in a new tab',
     'file-viewer.more-actions': 'More…',
+    'editor-menu.bold': 'Bold',
+    'editor-menu.italic': 'Italic',
+    'editor-menu.h1': 'Header level 1',
+    'editor-menu.h2': 'Header level 2',
+    'editor-menu.h3': 'Header level 3',
```

Comment:
Not a big deal, but in MS Words, Apple Pages, etc… they have always just called these "Heading 1", "Heading 2", etc… I'm a fan of "less is more". So if you wanna change the, I'd be very happy

Reply:
That's no problem, we can change that

Reply:
Updated

Reply:
`Header` or `Heading` for this? Currently we have committed `Header`.

Although this is probably not so important, but I really wanted to resolve this comment 😅

Reply:
The commit with the change was missed. I've rebased and it's all there now 😅